### PR TITLE
Update test configuration

### PR DESCRIPTION
**Pull request by cheungpat**

### CODE REVIEW

**Comment by oursky-travis** [comment]
@cheungpat, thanks for your PR! By analyzing the annotation information on this pull request, we identified @rickmak to be a potential reviewer

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -18,10 +19,15 @@ services:
       REDISTEST: redis://redis:6379
       PGHOST: db
       PGUSER: postgres
+      GOBIN: /go/bin
   db_cmd:
-    image: mdillon/postgis:9.5
```
Comment (rickmak) [inline docker-compose.dev.yml]: Why bring down to 9.4 here?

Comment (cheungpat) [inline docker-compose.dev.yml]: it shouldn't be here, thanks

**Comment by cheungpat** [comment]
updated @rickmak

**Pull request merged**